library/spdm_responder_conformance_test_lib/spdm_responder_test_7_measurements.c

```diff
@@ -1575,7 +1575,8 @@ void spdm_test_case_measurements_invalid_request (void *test_context)
         spdm_request_size = sizeof(spdm_get_measurements_request_t);
     }
 
-    for (index = 0; index < SPDM_MAX_SLOT_COUNT * 2; index++) {
+    /*the invalid sot_id is from 3 to 0x0E */
```

Comment:
We don't know which slot_ids, from 1 to 7, are valid or invalid.

Reply:
The valid slot can be detected via DIGESTS.

Reply:
Thanks. I have updated the comment.